### DISCUSSION TITLE
Is there any speaker encoder that outputs size 256?

### DISCUSSION THREAD

**Discussion by Rolun** [coqui-ai/TTS#2768] [General Q&A] [answered]
Hi all,

I'm looking for a speaker encoder that outputs a shape of 256 values as opposed to the 512 that the standard speaker encoder model comes with.
Is there any such model?

Many thanks,
Robert

**Comment by freds0** [comment, marked as answer]
[GE2E](https://arxiv.org/pdf/1710.10467.pdf) produces a 256 dimension speaker embedding. You can use this script to extract the embeddings:

[https://github.com/freds0/BSpeech-MOS-Prediction/tree/main/extract_emb](https://github.com/freds0/BSpeech-MOS-Prediction/tree/main/extract_emb)

But it is also available at [https://github.com/resemble-ai/Resemblyzer](https://github.com/resemble-ai/Resemblyzer)

**Comment by Rolun** [comment]
@freds0 thanks!
